_unittest/test_08_Primitives3D.py

```diff
@@ -1156,3 +1156,58 @@ def test_76_check_value_type(self):
         assert boolean2
         assert isinstance(resolve3, float)
         assert not boolean3
+
+    @pytest.mark.skipif(is_ironpython, reason="pytest is not supported with IronPython.")
+    @pyaedt_unittest_check_desktop_error
+    def test_77_create_helix(self):
+
+        udp1 = [0, 0, 0]
+        udp2 = [5, 0, 0]
+        udp3 = [10, 5, 0]
+        udp4 = [15, 3, 0]
+        polyline = self.aedtapp.modeler.create_polyline(
+            [udp1, udp2, udp3, udp4], cover_surface=False, name="helix_polyline"
+        )
+
+        helix_right_turn = self.aedtapp.modeler.create_helix(
+            polyline_name="helix_polyline",
+            position=[0, 0, 0],
+            x_start_dir=0,
+            y_start_dir=1.0,
+            z_start_dir=1.0,
+            num_thread=1,
+            right_hand=True,
+            radius_increment=0.0,
+            thread=1.0,
+        )
+
+        assert helix_right_turn.object_units == "mm"
+
+        # Test left turn without providing argument value for default parameters.
+        udp1 = [-45, 0, 0]
+        udp2 = [-50, 0, 0]
+        udp3 = [-105, 5, 0]
+        udp4 = [-110, 3, 0]
+        polyline_left = self.aedtapp.modeler.create_polyline(
+            [udp1, udp2, udp3, udp4], cover_surface=False, name="helix_polyline_left"
+        )
+
+        assert self.aedtapp.modeler.create_helix(
+            polyline_name="helix_polyline_left",
+            position=[0, 0, 0],
+            x_start_dir=1.0,
+            y_start_dir=1.0,
+            z_start_dir=1.0,
+            right_hand=False,
+        )
+
+        # Test that an exception is raised if the name of the polyline is not provided.
+        with pytest.raises(ValueError) as exc_info:
```

Comment:
You can use a workaround like this to be compatible with IronPython:
```
try:
   function_that_will_crash
except:
   assert True
else:
   assert False
```
So you don't need to skip the test in Ironpython

Reply:
@Alberto-DM Done. Thanks.